docker-entrypoint.sh

```diff
@@ -31,6 +31,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 			( set -x; ls -A; sleep 10 )
 		fi
 		tar cf - --one-file-system -C /usr/src/wordpress . | tar xf -
+		chown -R ${APACHE_RUN_USER:-www-data}:${APACHE_RUN_GROUP:-www-data} $PWD
```

Comment:
See https://github.com/docker-library/wordpress/pull/74 for where a `chown` similar to this one was previously removed very intentionally.

I think for this to work, we'll need to instead use the `--no-same-owner` and/or `--user`/`--group` flags of `tar` in order to control the ownership of the copied bits directly instead.